snapshots/input/multi-project/dump.svg

```diff
@@ -0,0 +1,300 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Could you provide some context for this SVG and the other one (how they're generated, why they're being checked in etc.)? I understand that they're showing the relations between different character ranges, monikers etc., but I didn't notice any doc comments or code related to these.